tests/e2e/test_browser.py

```
@@ -166,14 +166,41 @@ def do_login(page: Page, base_url: str) -> bool:
+            # Re-fill + re-click
+            try:
+                email_field.fill(ADMIN_EMAIL)
+                pw_field.fill(ADMIN_PASSWORD)
+                if btn.count() > 0:
+                    btn.click()
+                else:
+                    pw_field.press("Enter")
+                page.wait_for_load_state("domcontentloaded", timeout=15000)
+                page.wait_for_timeout(1500)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)